docs/user-docs/latest/media-library/introduction-to-media-library.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Introduction to the Media Library - Strapi User Guide
+description:
+canonicalUrl:
+---
+
+<!-- TODO: update SEO -->
+
+# Introduction to the Media Library
+
+The Media Library is a Strapi plugin that is always activated by default and cannot be deactivated. It is accessible both when the application is in a development and production environment.
+
+Administrators can access the Media Library from ![ML icon](../assets/icons/media_library.svg) _Media Library_ in the main navigation of the admin panel.
+
+<!-- TODO: replace screenshot -->
+![Media Library overview, annotated](../assets/media-library/media-library_overview.png)
+
+The Media Library displays all assets uploaded in the application, either via the Media Library itself or via the Content Manager when managing a media field. Assets uploaded to the Media Library can be inserted into content-types using the [Content Manager](/user-docs/latest/content-manager/writing-content.md#filling-up-fields).
+
+From the Media Library, it is possible to:
+
+- upload a new asset (1) (see [adding assets](/user-docs/latest/media-library/adding-assets.md)),
+- create a new folder (2) (see [organizing assets with folders](/user-docs/latest/media-library/organizing-assets-with-folders.md)),
+- sort the assets and folders (3) or set filters (4) to find assets more easily,
```

Comment:
I would say "sort the assets and folders (3) or set filters (4) to find assets **and folders** more easily"